.github/CUSTOM_AGENTS_INTEGRATION.md

```diff
@@ -0,0 +1,264 @@
+   - Standardizes test writing approach
+
+3. **TypeScript (typescript-5-es2022.instructions.md + typescript-mcp-expert.chatmode.md)**
+   - Entire codebase is TypeScript
+   - Using latest features (5.8.3)
+   - Type safety is paramount for library code
+
+4. **Research & Planning (research-technical-spike.chatmode.md, task-researcher.chatmode.md)**
+   - Complex monorepo with multiple packages
+   - Technical spikes are common for new features
+   - Systematic research improves decision-making
+
+5. **Issue Management (refine-issue.chatmode.md)**
+   - Helps maintain high-quality issue descriptions
+   - Ensures acceptance criteria are well-defined
+   - Improves team collaboration
+
+## Files Added
+
+```
+.github/
+├── COPILOT_CUSTOMIZATION.md          # User documentation
+├── CUSTOM_AGENTS_INTEGRATION.md      # This file (integration plan)
+├── agents/
+│   └── .gitkeep                      # Placeholder for future agents
+├── chatmodes/
+│   ├── accessibility.chatmode.md
+│   ├── refine-issue.chatmode.md
+│   ├── research-technical-spike.chatmode.md
+│   ├── task-researcher.chatmode.md
+│   └── typescript-mcp-expert.chatmode.md
+└── instructions/
+    ├── a11y.instructions.md
+    ├── playwright-typescript.instructions.md
+    └── typescript-5-es2022.instructions.md
+```
+
+## Usage Guidelines
+
+### For Developers
+
+1. **Instructions are automatic**: When you edit a file, relevant instructions are applied
+2. **Use chatmodes intentionally**: Reference them when working on specific tasks
+3. **Provide feedback**: Let the team know which customizations are helpful
+
+### For Contributors
+
+When contributing to the repository:
+- Follow the patterns in instructions files
+- Use accessibility chatmode when working on UI components
+- Use research chatmodes for technical spikes
+- Reference TypeScript expert for complex TypeScript patterns
+
+## Not Included (Evaluated and Rejected)
+
+We evaluated but did not include these from awesome-copilot:
+
+1. **copilot-thought-logging.instructions.md**
+   - Reason: Too verbose, would slow down workflow
+   - Creates unnecessary tracking files in workspace
+
+2. **instructions.instructions.md**
+   - Reason: Meta-instructions for creating instructions
+   - Not needed for day-to-day development
+
+3. **software-engineer-agent-v1.chatmode.md**
+   - Reason: Too opinionated and prescriptive
+   - Zero-confirmation policy conflicts with team collaboration
+   - Better to use task-specific chatmodes
+
+## Future Enhancements
+
+Potential additions as needs arise:
+
+1. **Custom Agents**
+   - Stencil-to-Lit migration agent
+   - Component generation agent
+   - Documentation generation agent
+
+2. **Additional Chatmodes**
+   - Performance optimization mode
+   - Bundle size analysis mode
+   - Breaking change detection mode
+
+3. **Package-Specific Instructions**
+   - Headless package instructions
+   - Quantic (Salesforce) specific guidelines
+   - React bindings conventions
+
+## Maintenance
```

Comment:
This section is interesting: these are new processes we'd have to insert into our flow.